src/i18n/en.js

```diff
@@ -12,6 +12,8 @@ export default {
   'btn-clear': 'Clear',
   'btn-select-image': 'Select an image',
   'btn-next': 'Next',
+  'publish-edit-title-placeholder': 'Give your story a title',
+  'publish-edit-error-msg': 'Enter story title',
```

Comment:
@medied @stephanebisson When you have time can you add the other translation?

Reply:
Here's the french version: 
  'publish-edit-title-placeholder': 'Donnez un title à votre histoire',
  'publish-edit-error-msg': 'Veuillez entrer un title pour votre histoire',

Reply:
I'm waiting for @medied then add them together.

Reply:
Portuguese:

```
 'publish-edit-title-placeholder': 'Dê um título à sua história',
 'publish-edit-error-msg': 'Insira o título da história',
```

Spanish:
```
 'publish-edit-title-placeholder': 'Ponle un titulo a tu historia',
 'publish-edit-error-msg': 'Agregar titulo',
```